File: triatlon-java/networking/src/main/java/triatlon/networking/dto/RefereeDTO.java
package triatlon.networking.dto;

import triatlon.model.activity.RaceType;

import java.io.Serializable;

public class RefereeDTO implements Serializable {

    private String firstName;
    private String lastName;
    private RaceType raceType;
    private String email;
    private String password;

    public RefereeDTO(String email, String password) {
        this.email = email;
        this.password = password;
    }

    public RefereeDTO(String firstName, String lastName, RaceType raceType, String email, String password) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.raceType = raceType;
        this.email = email;
        this.password = password;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public RaceType getRaceType() {
        return raceType;
    }

    public void setRaceType(RaceType raceType) {
        this.raceType = raceType;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: triatlon-cs/Networking/DTO/RefereeDto.cs
using System;
using Model.Activity;

namespace Networking.DTO
{
    [Serializable]
    public class RefereeDto
    {
        private string _firstName;
        private string _lastName;
        private RaceType _raceType;
        private string _email;
        private string _password;

        public RefereeDto(string email, string password)
        {
            _email = email;
            _password = password;
        }

        public RefereeDto(string firstName, string lastName, RaceType raceType, string email, string password)
        {
            _firstName = firstName;
            _lastName = lastName;
            _raceType = raceType;
            _email = email;
            _password = password;
        }

        public string FirstName
        {
            get => _firstName;
            set => _firstName = value;
        }
        
        public string LastName
        {
            get => _lastName;
            set => _lastName = value;
        }
        
        public RaceType RaceType
        {
            get => _raceType;
            set => _raceType = value;
        }
        
        public string Email
        {
            get => _email;
            set => _email = value;
        }
        
        public string Password
        {
            get => _password;
            set => _password = value;
        }
    }
}